src/components/Page-Header/index.tsx

```diff
@@ -58,6 +59,11 @@ export const PageHeader = () => {
           Let’s Talk
         </Button>
       </Nav>
+      <Nav className="Trk__dnone Trk__dnone--md Trk__dflex--sm ">
```

Comment:
Let us add tests in this page header that ensures all the items in the header are present when page is rendered